packages/reactive-element/src/test/reactive-element_test.ts

```diff
@@ -3075,4 +3075,95 @@ suite('ReactiveElement', () => {
       assert.equal(el2.attrValue, 'custom');
     });
   });
+
+  if (DEV_MODE) {
```

Comment:
This should be moved to `reactive-element_dev_mode_test`

Reply:
Ok, I moved these into that file.